src/main/java/sevensmurfs/rehub/model/message/request/TherapyResultRequest.java

```diff
@@ -0,0 +1,17 @@
+package sevensmurfs.rehub.model.message.request;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+import sevensmurfs.rehub.enums.TherapyStatus;
+
+@Data
+public class TherapyResultRequest {
+
+    @NotNull
+    private Long therapyId;
+
+    @NotNull
+    private String result;
+
+    private TherapyStatus status;
```

Comment:
Add @NotNull on TherapyStatus

Reply:
Why not @NotBlank?

Reply:
Because it is enumeration. @NotBlank only on String objects.